.github/workflows/goreleaser.yaml

```diff
@@ -53,10 +53,10 @@ jobs:
         run: go mod download
       -
         name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@v2
+        uses: goreleaser/goreleaser-action@v6
         with:
-          version: v1.25.1
-          args: release --debug --skip-validate
+          version: latest
```

Comment:
Could this get us into a pickle with future updates to goreleaser? Is there a version it could be pinned to?